### PR TITLE
Resolve stylelint issues from compilation

### LINKED ISSUE #161: Stylelint deprecated unit whitelist
Lots of these errors have been popping up.

```bash
'unit-whitelist' has been deprecated. Instead use 'unit-allowed-list'
```

Also solve remaining stylelint notices:
```bash
142:34	⚠  Unexpected "
    " (selector-descendant-combinator-no-non-space) [stylelint]
143:5	⚠  Expected indentation of 2 spaces (indentation) [stylelint]
144:7	⚠  Expected indentation of 4 spaces (indentation) [stylelint]
146:7	⚠  Expected indentation of 4 spaces (indentation) [stylelint]
147:7	⚠  Expected indentation of 4 spaces (indentation) [stylelint]
```

### PR DESCRIPTION
**Pull request by paulpopus**
Fixes #161

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -33,6 +33,6 @@
       "except": ["first-nested"],
       "ignore": ["after-comment"]
     } ],
-    "unit-whitelist": ["em", "rem", "%", "s", "deg", "ms", "vh"]
+    "unit-allowed-list": ["em", "rem", "%", "s", "deg", "ms", "vh", "px"]
```
Comment (cjstevens78) [inline .stylelintrc:36]: we took 'px' off the whitelist originally to force devs to use 'rem' or 'em' - theres' a mixin for px to rem conversion eg: @include rem(margin, 15);

**Review by cjstevens78 — CHANGES_REQUESTED** [review]

File: .stylelintrc (hunk 1 above)
Comment (paulpopus) [reply .stylelintrc]: Ok that's great, I agree with that, reversed the change.

File: assets/scss/lib/partials/_alert.scss [hunk 2]
```diff
@@ -139,12 +139,12 @@
     }
   }
 
-  .localgov_alert_banner--dismiss
-    a { // alternative styling for link button
```
[the remark this answers is not in the document]
Comment (paulpopus) [reply assets/scss/lib/partials/_alert.scss:143]: Makes sense, I've updated it.

**Review by cjstevens78 — APPROVED** [review]
Thanks !

**Pull request merged**